reagurk/src/components/footer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+
+export default function Footer(): JSX.Element {
+
+  return (
+    <>
+      <footer className="text-gray-400 bg-gray-800 body-font fixed
+             inset-x-0
+             bottom-0">
+        <div className="container px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+          <a className="flex title-font font-medium items-center md:justify-start justify-center text-white">
+            <img src="https://cdn.discordapp.com/emojis/792081164466126868.png?size=56" alt="gurkult logo" />

```

Comment:
The discord CDN should not be considered reliable as image hosting for us. If there is an image we need to use, we should have it locally.

Reply:
Resolved.